src/app/metadata/features/metadata-browser/metadata-browser.component.html

```diff
@@ -1,28 +1,66 @@
 <div class="grid grid-cols-12 grid-rows-none gap-5">
```

Comment:
```suggestion
<div class="grid grid-cols-12 grid-rows-none gap-x-5 gap-y-3">
```
Looks better when the chips are not that separated from the heading of the list